programs/ziti-edge-tunnel/include/windows/windows-service.h

```diff
@@ -0,0 +1,40 @@
+#ifndef ZITI_TUNNEL_SDK_C_WINDOWS_SERVICE_H
+#define ZITI_TUNNEL_SDK_C_WINDOWS_SERVICE_H
+
+#if _WIN32
+#include <stdbool.h>
+#include <windows.h>
+#include "uv.h"
+
+#define SVCNAME TEXT("ziti-edge-tunnel")
+#define DISPLAYSVCNAME TEXT("Ziti Desktop Edge Service")
+#define SVCDESCRIPTION TEXT("Access your Networks Secured by Ziti")
+
+int SvcStart(TCHAR *);
```

Comment:
should the Svc* functions be enclosed in the `extern "C" {` block, since they are defined in C?